manifests/plugin/dynflow.pp

```diff
@@ -28,6 +30,7 @@
   String $core_listen = $::foreman_proxy::plugin::dynflow::params::core_listen,
   Integer[0, 65535] $core_port = $::foreman_proxy::plugin::dynflow::params::core_port,
   Optional[Array[String]] $ssl_disabled_ciphers = $::foreman_proxy::plugin::dynflow::params::ssl_disabled_ciphers,
+  Optional[Array[String]] $tls_disabled_versions = $::foreman_proxy::plugin::dynflow::params::tls_disabled_versions,
```

Comment:
Should this be `Float` instead of `String`?

Reply:
I'd not really consider a version to be a float

Reply:
If anything I'd make it ```Optional[Array[Enum['1.1', '1.2']]]``` since 1.0 is always disabled anyway.

Reply:
1.3 is coming, too.   I can make it an enum or float if you really want, but if a user gives bad data it'll get logged and nothing bad will really happen.

Reply:
That's why I accepted String already :)

Reply:
Okay, lets stick with `String` 👍